marconi-chain-index/src/Marconi/ChainIndex/Indexers/EpochState.hs

```diff
@@ -340,6 +342,13 @@ instance ToJSON EpochNonceRow where
         , "blockNo" .= blockNo
         ]
 
+data LedgerStateFileMetadata = LedgerStateFileMetadata
+    { lsfMetaIsVolatile      :: !Bool
```

Comment:
Actually, this is determined by the filename of the LedgerState (as it has the `volatile` word in it). Are you suggesting that we instead add the current node tip in the filename of the LedgerState, and then automatically do the computation to determine if the LedgerState is volatile or not?

Reply:
I was suggesting to leave the volatile info out because the fact of whether a ledger state is volatile or not changes in time (when current tip is more than security parameter blocks away from the slot of the ledger state). So ledger states that once were volatile become non-volatile as the tip advances, maybe there is some work to be saved by not deleting them.

Reply:
See 479869bf8868db054f4c794b413eea492465fbaa